experimental/dds/tree2/src/shared-tree-core/editManager.ts

```diff
@@ -201,6 +218,77 @@ export class EditManager<
 		});
 	}
 
+	/**
+	 * Makes the given branch known to the `EditManager` as a revertible branch. This ensures that the `EditManager` will
+	 * keep track of the oldest revertible sequence ID on this branch and will not trim any commits after it.
+	 *
+	 * @privateRemarks
+	 * TODO: this is not currently used and is included for documentation purposes
```

Comment:
I decided to keep this code here cause it seems like it'll need to be addressed soon. I can def remove it though if it seems messy or anything.

Reply:
Yeah, I'll just throw it out